src/styles/globals.scss

```diff
@@ -9,6 +9,12 @@ body {
   scroll-padding-top: 4.25rem;
 }
 
+/* This prevents the background from scrolling while a modal is open. */
+/* stylelint-disable-next-line selector-no-qualifying-type */
+body:has(dialog[open]) {
+  overflow: hidden;
```

Comment:
low priority but hiding the scrollbar on windows causes a layout shift when the dialog opens


https://github.com/acmucsd/membership-portal-ui-v2/assets/22133785/e5453136-8f7c-4255-8300-01821d3debf9

it's probably more trouble than it's worth trying to fix it, though. one idea is to do

```css
html {
  overflow-y: scroll; /* force scrollbar to take up space */
}
body {
  display: fixed; /* disable scrolling */
}
```

but `overflow-y: scroll;` will make a scrollbar that takes up space appear on Macs even when it normally wouldn't